src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,48 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.InvalidUserDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private final int minPassSize = 6;
```

Comment:
Please follow the constants naming rules
```suggestion
    public static final int MIN_PASSWORD_SIZE = 6;
```

Reply:
я делал всё по правилам, но мейвен не принимал константы. сначала просил убрать статик, а потом не принимал названия с больших букв